src/middleware/wrapSubmissionPromise.js

```diff
@@ -12,17 +12,30 @@ const wrapSubmissionPromise = withPropsOnChange(
       onSubmit: (...args) => {
         const result = onSubmit(...args)
         if (!isPromise(result)) return Promise.resolve(result)
-        return result.catch(err => {
-          const messages = getOr({}, 'errors', err)
-          const submissionError = new SubmissionError(messages)
-          
-          // Retain metadata (e.g., status code) about the original error
-          submissionError.meta = { error: err }
-          throw submissionError
-        })
+        return result.catch(mapSubmissionError)
       }
     }
   }
 )
 
+function mapSubmissionError (error) {
+  const messages = getErrorMessages(error)
+  const submissionError = new SubmissionError(messages)
+  
+  // Retain metadata (e.g., status code) about the original error
+  submissionError.meta = { error }
+  throw submissionError
+}
+
+// Checks for an error that matches LPL's default standard for mapping error messages (HttpError).
+// Else defaults to the standard error API and maps to a redux-form "form-wide" error key.
+function getErrorMessages (err) {
+  const messages = get('errors', err)
+  
+  if (messages) return messages
+  
+  const _error = get('message', err)
```

Comment:
This could be a chance to be a little more self-documenting (at the expense of the shorthand):
`return formWideError ? { _error: formWideError  } : {}`